### ISSUE TITLE
Can't find lib.d.ts path

### ISSUE THREAD

**Issue by cotarou** [ivogabe/gulp-typescript#62]
I got a following message when execute gulp command after install gulp-typescript.

```sh
Error: ENOENT, no such file or directory '/Users/me/projects/xx/tmp/node_modules/gulp-typescript/node_modules/typescript/bin/lib.d.ts'
    at Object.fs.openSync (fs.js:439:18)
    at Object.fs.readFileSync (fs.js:290:15)
    at Function.Host.initLibDefault (/Users/me/projects/xx/tmp/node_modules/gulp-typescript/release/host.js:14:26)
    at Object.<anonymous> (/Users/me/projects/xx/tmp/node_modules/gulp-typescript/release/host.js:79:6)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
```

then i used next gulp task.

```sh
var typescript = require('gulp-typescript');
var concat = require('gulp-concat');

var typescriptProject = typescript.createProject({
  target: "ES5",
  removeComments: true,
  sortOutput: true
});
gulp.task('default', function() {
  gulp.src(['app/scripts/*.ts'])
  .pipe(typescript(typescriptProject))
  .js
  .pipe(concat("main.js"))
});
```

so i open a host.js file in node_modules/gulp-typescript/release. and i read at 14 line then i execute npm install command at node_modules/gulp-typescript.
after the npm install i succeeded the build.

Is this a my mistake for install of gulp-typescript plugin ?

**Comment by bryanerayner** [comment]
I'm encountering the same problem, though I haven't been able to fix it quite yet.

**Comment by bryanerayner** [comment]
Resolved by CDing into node_modules/grunt-typescript and running
    npm install

Is there an issue which prevents this from happening by default?

**Comment by whitneyit** [comment]
This was resolved in #59. Just waiting on an updated release to be pushed to npm

**Comment by cotarou** [comment]
:+1:  thanks a lot.

**Comment by ivogabe** [comment]
Duplicate of #59 indeed
